packages/sdk/src/addresses/index.ts

```diff
@@ -119,25 +119,31 @@ export async function getAddresses({
 export function getAccountDataFromHdNode({
   hdNode,
   format = "legacy",
-  network = "testnet"
+  network = "testnet",
+  accountIndex = 0,
+  index = 0
```

Comment:
rename as per [BIP44 specification](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels)

```suggestion
  account = 0,
  addressIndex = 0
```